tests/roots/test-root/includes.txt

```diff
@@ -3,6 +3,7 @@ Testing downloadable files
 
 Download :download:`img.png` here.
 Download :download:`this <subdir/img.png>` there.
+Download :download:`file with special characters <file_with_special_#?_chars.xyz>`.
```

Comment:
Sadly, this filename was invalid on Windows platform. Could you rename it please?

Reply:
I dropped the question mark. Hopefully works now.